tests/copyright_check/config.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import sys
+import os
+
+
+debug_logs = False
+print_logs = sys.stderr if debug_logs else open("/dev/null", "w")
```

Comment:
Using `/dev/null` creates a dependency on Unix-like platform which is not really needed.

Using something like in the suggestion would allow you to not define the implicit dependency

```suggestion
print_logs = sys.stderr if debug_logs else open(os.devnull, "w")
```